apps/ledger-live-desktop/src/internal/index.js

```diff
@@ -12,11 +11,13 @@ import LoggerTransport from "~/logger/logger-transport-internal";
 import { executeCommand, unsubscribeCommand, unsubscribeAllCommands } from "./commandHandler";
 import sentry, { setTags } from "~/sentry/internal";
 
+let unsubscribeSentry = () => {};
+
 process.on("exit", () => {
   logger.debug("exiting process, unsubscribing all...");
   unsubscribeSetup();
   unsubscribeAllCommands();
-  getSentryIfAvailable()?.close(2000);
+  if (unsubscribeSentry) unsubscribeSentry();
```

Comment:
We could remove the check here but I suppose it's also fine with it
```suggestion
  unsubscribeSentry();
```